### PR TITLE
Styling updates to the user guide

### PR DESCRIPTION
**Pull request by scoyle391**
This PR addresses styling issues such as markdown errors and grammatical changes.

### CODE REVIEW

**Comment by scoyle391** [comment]
Ugly link, not broken link.

File: user-guide/bare-metal.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ spec:
...
 ```
 
-This configuration does not require an Ambassador service be defined so you can remove that service if you have defined one.
+This configuration does not require a defined Ambassador service, so you can remove that service if you have defined one.
 
-**Note:** Before configuring Ambassador with this method, consider some of the functionality that is lost by bypassing the Kubernetes service including only having one Ambassador able to bind to port 8080 or 8443 per node and losing any load balancing that is typically performed by Kubernetes services. 
+**Note:** Before configuring Ambassador with this method, consider some of the functionality that is lost by bypassing the Kubernetes service including only having one Ambassador able to bind to port 8080 or 8443 per node and losing any load balancing that is typically performed by Kubernetes services.
 
-Join our [Slack channel](https://d6e.co/slack) to ask any questions you have regarding running Ambassador on a bare metal installation.
+Join our [Slack channel](https://d6e.co/slack) to ask any questions you have regarding running Ambassador on a Bare Metal installation.
```
Comment (khussey) [inline user-guide/bare-metal.md]: Why are we capitalizing "bare metal" here?

Comment (scoyle391) [reply user-guide/bare-metal.md]: addressed!

**Comment by scoyle391** [comment]
blocked by https://github.com/datawire/getambassador.io/pull/184

**Pull request merged**